pyanaconda/installation.py

```diff
@@ -166,6 +166,18 @@ def _prepare_configuration(self, payload, ksdata):
                 (overwrite, )
             ))
             configuration_queue.append(network_config)
+        # schedule hostname configuration (if required)
+        elif conf.system.can_change_hostname:
+            hostname_config = TaskQueue(
+                "Hostname configuration",
+                _("Writing hostname configuration")
+            )
+            hostname_config.append(Task(
+                "Hostname configuration",
+                network.network.write_configuration_hostname,
```

Comment:
Anaconda crashes here, there should be `network.write_configuration_hostname`

Reply:
https://github.com/rhinstaller/anaconda/pull/4870/files#diff-335d1ad6ab4d87d2febae93c61ef93bb406a9913b4fef5503e337481c28ee697R57
https://github.com/rhinstaller/anaconda/pull/4870/files#diff-335d1ad6ab4d87d2febae93c61ef93bb406a9913b4fef5503e337481c28ee697R279

Reply:
That's exactly what was done. First, the operation is divided into two functions, and then they are called in turn depending on can_change. Moreover, the formulation of the name configuration task comes just after the network configuration task.

Reply:
> Anaconda crashes here, there should be `network.write_configuration_hostname`

fixed typo :-)